test/test-asd.js

```diff
@@ -0,0 +1,261 @@
+// should test the following:
+// 1. It should grab the test using 'grabTest()' from the selected project
+// 2. It should run the test using 'runTest()' from the selected project
+// 3. might have to create own pass /fail object structure for asd
+
+/* global describe it expect before beforeEach afterEach */
+require('mocha');
+require('should');
+const fs = require('fs');
+const rimraf = require('rimraf');
+const expect = require('chai').expect;
+const proxyquire = require('proxyquire');
+const changeCase = require('change-case');
+
+const fakeHelpers = require('./helpers/fakeHelpers');
+
+const {
+  dummySessionAsd,
+} = require('./helpers/dummyData');
+
+const projects = proxyquire('../controller/projects', {
+  './env': {
+    home: fakeHelpers.home
+  }
+});
+
+const test = proxyquire('../controller/test', {
+  './helpers': fakeHelpers,
+  './github': fakeHelpers,
+  './env': {
+    home: fakeHelpers.home
+  }
+});
+
+const projectsDirectory = './test/files/environment/projects';
+
+describe('test', function () {
+  before(function (done) {
+    rimraf(projectsDirectory, () => done());
+  });
+
+  afterEach(function () {
+    if (console.log.restore) console.log.restore();
+  });
+
+  // Should install tests for the selected project
+  describe.only('#grabTests()', function () {
```

Comment:
Not sure if this was a mistake, but `only` will skip all the rest of the tests
```suggestion
  describe('#grabTests()', function () {
```

Reply:
@harveysanders it was a mistake , I had it for testing purposes.